.github/workflows/rust_builder.yml

```diff
@@ -29,6 +29,8 @@ jobs:
         exclude:
           - scheduled: true
             toolchain_version: stable
+        protobuf_compiler:
```

Comment:
Hey @atanmarko, what do you think about moving the `protobuf_compiler` to the `env` section (starts on the [line 11](https://github.com/topos-network/topos-ci-docker/blob/2f784aa8bd3cf708e024eb592d8d9b421639a7a2/.github/workflows/rust_builder.yml#L11))?